File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Microtime.java
package com.onevgo.functions;

import java.time.Instant;

public class Microtime {
    public static String microtime() {
        Instant now = Instant.now();
        return String.format("%.8f %d", now.getNano() * 1.0 / 1000_000_000, now.getEpochSecond());
    }

    public static double microtime(boolean getAsFloat) {
        if (!getAsFloat) {
            throw new IllegalArgumentException("getAsFloat");
        }
        Instant now = Instant.now();
        return now.getEpochSecond() + now.getNano() * 1.0 / 1000_000_000;
    }

    public static void main(String[] args) {
        System.out.println(microtime());
        System.out.printf("%f", microtime(true));
    }
}

File: 01_Language/01_Functions/python/microtime.py
# coding: utf-8

import time
import math

def microtime(get_as_float=False):
    now = time.time()
    return now if get_as_float else '%f %d' % math.modf(now)

if __name__ == '__main__':
    print(microtime())
    print(microtime(True))